### PR TITLE
Revert logit

### PR DESCRIPTION
**Pull request by atte-wsp**
Undo: February 7 "Avoid side effects when calculating parts of utility". This merge causes the full model system to produce erroneous assignment results. I haven't yet figured out the reason, but would merge this to the master, as I think it should be functional for testing purposes.

### CODE REVIEW

**Comment by zptro** [comment]
@atte-wsp, could you fix this and merge asap?

File: Scripts/models/logit.py [hunk 1]
```diff
@@ -62,19 +62,19 @@ def _calc_dest_util(self, mode, impedance):
...
-        utility += self._add_sec_zone_util(utility, b["attraction"], orig, dest)
-        utility += self._add_impedance(utility, impedance, b["impedance"])
+        self._add_sec_zone_util(utility, b["attraction"], orig, dest)
+        self._add_impedance(utility, impedance, b["impedance"])
         dest_exps = numpy.exp(utility)
         size = numpy.zeros_like(utility)
         size += self._add_sec_zone_util(size, b["size"])
         impedance["size"] = size
-        dest_exps *= self._add_log_impedance(dest_exps, impedance, b["log"])
+        self._add_sec_zone_util(size, b["size"])
```
[the remark this answers is not in the document]
Comment (atte-wsp) [reply Scripts/models/logit.py]: That's correct! There was simple copy / paste mistake. I should have been more careful before commit. Now corrected.

File: Scripts/models/logit.py [hunk 2]
```diff
@@ -62,19 +62,19 @@ def _calc_dest_util(self, mode, impedance):
     def _calc_sec_dest_util(self, mode, impedance, orig, dest):
         b = self.dest_choice_param[mode]
         utility = numpy.zeros_like(next(iter(impedance.values())), self.dtype)
-        utility += self._add_sec_zone_util(utility, b["attraction"], orig, dest)
-        utility += self._add_impedance(utility, impedance, b["impedance"])
+        self._add_sec_zone_util(utility, b["attraction"], orig, dest)
+        self._add_impedance(utility, impedance, b["impedance"])
         dest_exps = numpy.exp(utility)
         size = numpy.zeros_like(utility)
         size += self._add_sec_zone_util(size, b["size"])
```
Comment (zptro) [inline Scripts/models/logit.py]: This is still done in Johanna's style?

Comment (zptro) [inline Scripts/models/logit.py]: This is still done in Johanna's style though?

Comment (atte-wsp) [reply Scripts/models/logit.py]: Checked this one from commits and that's correct. I already merged this pull request, so it seems that I need to do another pull request to fix this.

**Pull request merged**